README.md

```diff
@@ -27,10 +29,16 @@ You can visit the book page [here](https://datasciencejuliahackers.com/)
 
 * [Nineth chapter](https://datasciencejuliahackers.com/09_optimal_pricing.jl.html): we solve a problem of optimal pricing optimization using a bayesian point of view.
 
+### Part III: Machine Learning
+* Work in progress
+
+### Part IV: Deep Learning
 * [Tenth chapter](https://datasciencejuliahackers.com/10_bees_vs_wasps.jl.html): in this chapter we give an overview of Machine Learning and we implement a simple convolutional neural network to solve a bees vs. wasps classification.
```

Comment:
Capitalization after semicolon

Reply:
Overview of Machine Learning and implementation of a simple convolutional neural network that is able to discriminate between pictures of bees and wasps.